### PR TITLE
Add Rubocop

### PR DESCRIPTION
**Pull request by DirtyF**
Adopt same rules as Jekyll implemented from https://github.com/styleguide/ruby/

### CODE REVIEW

**Comment by DirtyF** [comment]
@nhoizey I didn't test the plugin, please confirm that it does not break anything.

There are a few warning left, that should be adressed in a separate issue.

File: lib/jekyll/cloudinary/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Jekyll
   module Cloudinary
-    VERSION = "1.2.15"
+    VERSION = "1.2.15".freeze
```
Comment (nhoizey) [inline lib/jekyll/cloudinary/version.rb]: What does this "freeze" mean?

File: lib/jekyll/cloudinary.rb [hunk 2]
```diff
@@ -11,161 +10,196 @@ def initialize(tag_name, markup, tokens)
...
+            .match(rendered_markup)
+        unless markup
+          Jekyll.logger.abort_with("[Cloudinary]", "Can't read this tag: #{@markup}")
         end
 
         image_src = markup[:image_src]
 
         # Build source image URL
-        is_image_path_absolute = /^\/.*$/.match(image_src)
+        is_image_path_absolute = %r!/^\/.*$/!.match(image_src)
```
Comment (nhoizey) [inline lib/jekyll/cloudinary.rb]: What does it?

File: lib/jekyll/cloudinary.rb [hunk 3]
```diff
@@ -11,161 +10,196 @@ def initialize(tag_name, markup, tokens)
...
         end
 
         # Get source image natural width
         if File.exist?(image_path)
-          image = Magick::Image::read(image_path).first
+          image = Magick::Image.read(image_path).first
           natural_width = image.columns
-          fallback_url = "https://res.cloudinary.com/#{settings['cloud_name']}/image/fetch/c_scale,w_#{natural_width},q_auto,f_auto/#{image_url}"
+          fallback_url = "https://res.cloudinary.com/\
+          #{settings["cloud_name"]}/image/fetch/c_scale,w_\
```
Comment (nhoizey) [inline lib/jekyll/cloudinary.rb]: I find lisibility suffers from this, maybe it would be better with indentations?

**Pull request closed without merging**